relayer/chains/evm/client.go

```diff
@@ -9,55 +9,43 @@ import (
 	ethereum "github.com/ethereum/go-ethereum"
 	"github.com/ethereum/go-ethereum/accounts/abi/bind"
 	ethTypes "github.com/ethereum/go-ethereum/core/types"
-	"github.com/ethereum/go-ethereum/rpc"
 	bridgeContract "github.com/icon-project/centralized-relay/relayer/chains/evm/abi"
-	"github.com/icon-project/centralized-relay/relayer/chains/evm/types"
-	providerTypes "github.com/icon-project/centralized-relay/relayer/types"
-	"github.com/pkg/errors"
 	"go.uber.org/zap"
 
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/ethclient"
 )
 
 const (
-	RPCCallRetry           = 5
-	MaxTxFixtures          = 5
-	DefaultMinedTimeout    = time.Minute * 5
-	DefaultPollingInterval = time.Second * 5
-	MaximumWaitTry         = 60
+	DefaultPollingInterval = time.Second * 30
```

Comment:
These constant values should be better handled from chain configuration because these are different for different chains.

Reply:
The idea is to have less config options as possible.
Having so much config option is prone to errors and maintenance burden.

Reply:
And these options are very carefully choosen for default values.

Having other values may have other consequences.